ethstorage/p2p/protocol/syncclient.go

```diff
@@ -942,10 +942,10 @@ func (s *SyncClient) OnBlobsByRange(res *blobsByRangeResponse) {
 	}
 	s.lock.Lock()
 	res.req.subTask.task.healTask.insert(missing)
-	if max == res.req.subTask.Last-1 {
+	if last == res.req.subTask.Last-1 {
```

Comment:
What is the bug that you were trying to fix (mentioned on the title)?

Reply:
just because goland give a warning